Identification-and-Assessment.tex

```diff
@@ -30,49 +30,88 @@ \subsection{Create Risk Item}
-	\item Cost or Schedule Impact Description
+	% \item Cost or Schedule Impact Description. PJM: See above, I think we always need a description any time we specify an impact.
 	\item Related Actions
 \end{itemize}
 
 Verify assessment in the automatically generated fields under Analyze Risk Score.
 
+When analyzing a risk that is the parent of $N$ child risks:
+\begin{itemize}
+	\item First analyze the children separately, i.e. estimating Residual Probability $P_k$, Likely Cost $C_k$, and Likely Delay $S_k$ for the $k$-th risk, and repeating for all $k$ in the range $(1 \ldots N)$.
+	\item For each risk, calculate the Cost Exposure $E_k = C_k \times P_k$.
+	\item The combined Cost Exposure for the parent risk is just the sum of the Cost Exposures of its children, $E_{\rm total} = \sum_{k=1}^N E_k$.
+	\item The Likely Cost of the parent risk is the sum of the Likely Costs of its children, $C_{\rm total} = \sum_{k=1}^N C_k$.
+	\item The Residual Probability of the parent risk is its Cost Exposure divided by its Likely Cost, $P_{\rm total} = E_{\rm total} / C_{\rm total}$.
+	This is just the cost-weighted average of the child risks' Residual Probabilities, and is the most meaningful definition of probability for parent risks.
+	\item Other quantitative attributes of parent risks can be computed by simple summation (e.g. Minimum Cost, Maximum Delay, Schedule Exposure, etc).
+	\item Pre-response quantities of parent risks, such as Impact Scores, can be reverse engineered from their Residual Probabilities.
+\end{itemize}
+
 \subsection{Continuous Review and Updates to Active Risks}
 
-TBD for RROB monthly meetings.
+Risk Owners flag issues or proposed changes to Active Risks for review at an RROB monthly meeting.
+
 Managing departments should review risks on a frequency based on severity.
 
-\subsection{Realized Risk Trigger}
+In the (May) Annual Scrub, Level 3 Team Leaders are invited to review, in the Scrub Sandbox, pertinent fields of risks that their team is affected by and which they are expected to help respond to.
+Modifications they propose are reviewed by the RROB and implemented in the Risk Register by the Risk Owners in the post-scrub implementation phase (June).
+
+\subsection{Addressing Realized Risks}
+
+Triggering a Realized Risk:
+Review of all Response Plans and Actions by Risk Owner, and follow-up review with department management.
+Risk Owner proposes change of Status from Active to Realized.
+Review with RROB is required if severity is above threshold (determined by RROB Chair).
+
+Addressing a Realized Risk:
+Risk Owner works with department management to submit a Request Beyond Target (RBT) for additional resources needed to address the risk (see RTN-005 for the RBT process).
+RBT is reviewed by RDO, which works with department for any needed escalation to NOIRLab, SLAC or the funding agencies or Resource Forum.
```

Comment:
What is "Resource Forum" ?

Reply:
Follow up from slack discussion.  Add a sentence making it clear that if a parent risk is broken down into child risks, then child risks must be non-overlapping, fully independent and fully cover the parent risk. This way the assessment  in terms of Cost Exposure, etc, is just a simple linear sum of the child risks

Reply:
Resource Forum = US agencies plus international in-kind contribution program's funding agencies. So its a bit redundant with "funding agencies" but not incorrect